src/hx711-element.ts

```diff
@@ -0,0 +1,491 @@
+        text {
+          fill: #dfe2e5;
+          user-select: none;
+        }
+      </style>
+      <svg
+        width="34mm"
+        height="20mm"
+        viewBox="0 0 340 200"
+        version="1.1"
+        xmlns="http://www.w3.org/2000/svg"
+        xmlns:xlink="http://www.w3.org/1999/xlink"
+      >
+        <title>HX711</title>
+        <defs>
+          <radialGradient
+            cx="45.4232183%"
+            cy="37.4460791%"
+            fx="45.4232183%"
+            fy="37.4460791%"
+            r="71.4089358%"
+            gradientTransform="translate(0.454232,0.374461),scale(0.588235,1.000000),rotate(88.619767),translate(-0.454232,-0.374461)"
+            id="radialGradient-1"
+          >
+            <stop stop-color="#3FA35A" offset="0%"></stop>
+            <stop stop-color="#216B34" offset="100%"></stop>
+          </radialGradient>
+          <path d="M0,0 L340,0 L340,200 L0,200 L0,0 Z" id="path-2"></path>
+          <mask
+            id="mask-3"
+            maskContentUnits="userSpaceOnUse"
+            maskUnits="objectBoundingBox"
+            x="0"
+            y="0"
+            width="340"
+            height="200"
+            fill="white"
+          >
+            <use xlink:href="#path-2"></use>
+          </mask>
+          <linearGradient x1="0%" y1="50%" x2="100%" y2="50%" id="linearGradient-4">
+            <stop stop-color="#767676" offset="0%"></stop>
+            <stop stop-color="#FFFEDF" offset="100%"></stop>
+          </linearGradient>
+          <radialGradient cx="50%" cy="50%" fx="50%" fy="50%" r="37.7940956%" id="radialGradient-5">
+            <stop stop-color="#FFFFFF" offset="0%"></stop>
+            <stop stop-color="#000000" offset="100%"></stop>
+          </radialGradient>
+        </defs>
+        <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+          <g
+            id="board"
+            fill="url(#radialGradient-1)"
+            stroke="#ADA216"
+            stroke-dasharray="1,1"
+            stroke-opacity="0.536433565"
+            stroke-width="4"
+          >
+            <use mask="url(#mask-3)" xlink:href="#path-2"></use>
+          </g>
+          <g
+            id="copper"
+            transform="translate(46.000000, 44.000000)"
+            stroke="#44E06D"
+            stroke-linecap="round"
+            stroke-width="6"
+          >
+            <path d="M143.5,1.5 L203.5,1.5" id="Line"></path>
+            <path d="M205,3 L240,24" id="Line"></path>
+            <path d="M142.5,51.5 L202.5,51.5" id="Line"></path>
+            <path d="M16.5,63.5 L76.5,63.5" id="Line"></path>
+            <path d="M48.5,126.5 L189,126.5" id="Line"></path>
+            <path d="M48.5,137.5 L189,137.5" id="Line"></path>
+            <path d="M16.5,63.5 L0,51" id="Line"></path>
+            <path d="M204,53 L239,74" id="Line"></path>
+            <path d="M143.5,39.5 L203.5,39.5" id="Line"></path>
+            <path d="M205,40 L235,48" id="Line"></path>
```

Comment:
The `id` attributes can be remove altogether (as far as I can tell, they are unused, and they way SVG works, if you assign an `id`, it should be unique across the document)